tspserver/src/main/java/com/komiwojazer/bean/User.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.komiwojazer.bean;
```

Comment:
Jak już tak ładnie piszemy wszystko po angielsku to też bym nazwę paczki zmienił na np. `com.tsp.bean`.

Reply:
zrobione